ogr/services/github.py

```diff
@@ -174,9 +174,15 @@ def pr_comment(
         body: str,
         commit: str = None,
         filename: str = None,
-        row: int = None,
+        position: int = None,
```

Comment:
please don't rename the argument because it wouldn't no longer work with other git projects:

https://github.com/packit-service/ogr/blob/master/ogr/services/pagure.py#L132